objects.js

```diff
@@ -4827,6 +4827,10 @@ SpriteMorph.prototype.mouseDownLeft = function () {
     return this.receiveUserInteraction('pressed');
 };
 
+SpriteMorph.prototype.mouseScroll = function (y) {
+    return this.receiveUserInteraction('scrolled-' + (y > 0 ? 'up' : 'down'));
```

Comment:
Hmm, this feels weird on a Mac nowadays, where they've mirrored the direction. But it's nice to play with this!